nncf/experimental/tensor/functions/torch_numeric.py

```diff
@@ -242,3 +242,34 @@ def _(
 @numeric.size.register(torch.Tensor)
 def _(a: torch.Tensor) -> int:
     return torch.numel(a)
+
+
+@numeric.quantile.register(torch.Tensor)
+def _(
+    a: torch.Tensor,
+    q: float,
+    axis: Optional[Union[int, Tuple[int, ...]]] = None,
+    method: Optional[str] = "linear",
+    keepdims: bool = False,
+) -> float:
+    return torch.quantile(a, q, dim=axis, interpolation=method, keepdim=keepdims)
```

Comment:
torch.quantile is not reliable, please see implementation https://github.com/openvinotoolkit/nncf/pull/2279/files#diff-48a31470e6f5691c217d11e1f28cfcf5dfa25ffdd047ca71353e0ec829b9485eR200-R219